pkg/helm/actions/config.go

```diff
@@ -35,6 +35,12 @@ func GetActionConfigurations(host, ns, token string, transport *http.RoundTrippe
 		},
 		Transport: transport,
 	}
+	if inCluster {
```

Comment:
Don't create a new flag for this. We know what CA to use already based on the logic in main.go